### PR TITLE
Added bot_help.py

### PR DESCRIPTION
**Pull request by Kurolox**
Also modified a small problem in bot.py that could give problems when launching

### CODE REVIEW

**Comment by RonakR** [comment]
DO NOT MERGE. Not ready.

**Comment by Kurolox** [comment]
Okay, it's done.

File: triviabot/mod_help.py [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
-    "command2": "command2-description"
-}
+command_dict = {}
 
 
 def on_load(bot):
     bot.add_command('helpme', helpme)
 
 
 def on_exit(bot):
```
Comment (SaschaMann) [inline triviabot/mod_help.py]: This is currently not implemented in this bot and thus never called. Can leave it in for the future though.

File: triviabot/mod_help.py [hunk 2]
```diff
@@ -1,30 +1,37 @@
...
-    bot.del_command('help')
+    bot.del_command('helpme')
```
Comment (SaschaMann) [inline triviabot/mod_help.py]: For consistency with most other bots in our channel, could you change this to just 'help' please?

File: triviabot/mod_help.py [hunk 3]
```diff
@@ -1,30 +1,37 @@
...
     # Sends a description of each command
     for command in command_dict:
         message += '\n' + command + ": " + command_dict.get(command)
     # Copyright and stuff
     message += '\n\nThis product is not endorsed, certified or otherwise approved in any way by Riot Games, Inc. or any of its affiliates.'
-    message += '\nCreated by SasCologne, boreeas, and FalconSpy. Hosted by FalconSpy.'
     # Send message to user
-    bot.send_msg(user, message)
+    nickname = separate_name(user)
+    print(nickname)
```
Comment (SaschaMann) [inline triviabot/mod_help.py]: Leftover print from debugging

File: triviabot/mod_help.py [hunk 4]
```diff
@@ -1,30 +1,37 @@
...
+    bot.send_msg(nickname[0], message)
+
+
+def command_help(command, helptext):
+    """Generate a entry in the command_dict dictionary that gives info about the command and it's usage when "helpme" is used."""
+    command_dict[command] = helptext
+
+command_help("helpme", "displays this message.")
```
Comment (SaschaMann) [inline triviabot/mod_help.py]: this is never called. it should be added to `on_load()`

File: triviabot/mod_help.py [hunk 5]
```diff
@@ -1,30 +1,37 @@
...
     message = '\nThe following commands are available: '
     # Generates a list of available commands
     command_list = []
     for command in command_dict:
         command_list.append(command)
-    message += command_list[:-2].split(", ") + ' and ' + command_list[-1]
+        message += command + ' '
     # Sends a description of each command
     for command in command_dict:
         message += '\n' + command + ": " + command_dict.get(command)
```
Comment (SaschaMann) [inline triviabot/mod_help.py]: for consistency, please change the double quotes `"` to single quotes `'`

File: triviabot/mod_help.py (hunk 4 above)
Comment (SaschaMann) [inline triviabot/mod_help.py]: for consistency, please change the double quotes `"` to single quotes `'`

File: triviabot/mod_help.py [hunk 6]
```diff
@@ -1,30 +1,37 @@
+from utilities import separate_name
+
 global command_dict
```
Comment (SaschaMann) [inline triviabot/mod_help.py]: the global keyword is only used to change variables in a local context, so it's not needed here. this line can be removed, line 4 is sufficient.

**Comment by Kurolox** [comment]
Everything corrected

**Pull request merged**